packages/terra-site/webpack.config.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
                     'iOS >= 8',
                   ],
                 }),
-                CustomProperties({ preserve: true }),
+                CustomProperties({ preserve: true, warnings: false }),
```

Comment:
what warnings are you getting?

Reply:
![image](https://user-images.githubusercontent.com/535099/29716501-b687ac24-8979-11e7-852d-3bc7f0688919.png)

Reply:
There are warnings when applying CSS custom properties under a selector that is not `:root`.  When we add a theme to the `$bundled-theme` SCSS variable, it will apply the CSS custom properties under a class selector if the theme if a theme in `$bundled-theme` is also not the `$default-theme`. 

This isn't an issue other than the warning are concerning if people are seeing them and not sure why they are generated. I'm fine with keeping the warnings as it would be nice to see if there were other issues with custom properties, but I don't foresee us running into other issues that will cause warning so I'm also fine disabling warning for the custom properties plugin.

Reply:
it's the change I added in the button theme branch. More info: https://github.com/MoOx/postcss-cssnext/issues/186